lib/Fhp/Parser/MT940.php

```diff
@@ -187,7 +187,7 @@ protected function parseDescription($descr)
         } else {
             $lastType = null;
             foreach ($descriptionLines as $line) {
-                if (strlen($line) > 5 && $line[4] === '+') {
+                if (strlen($line) > 5 && $line[4] === '+' || is_null($lastType) && $line === 'EREF+') {
```

Comment:
Please add parentheses so that the reader does not need to guess the precedence of `&&` vs `||`.